src/mdbExtensionController.ts

```diff
@@ -310,30 +311,34 @@ export default class MDBExtensionController implements vscode.Disposable {
     );
     this.registerCommand(
       EXTENSION_COMMANDS.CONNECT_WITH_PARTICIPANT,
-      (id?: string) =>
-        this._participantController.connectWithParticipant(
-          id ? decodeURIComponent(id) : id
-        )
+      (data: any) => {
```

Comment:
```suggestion
      (data: { id?: string; command?: string }) => {
```

Reply:
Thanks for catching, I'm thinking we can add some shared types here